docs/APISpec.md

```diff
@@ -0,0 +1,185 @@
+# APISpec
+
+## Usage
+
+All requests for a card are made to `/api/testaustime` and parameters are passed as query strings.
+
+## Endpoints
+The API provides 17 routes:
```

Comment:
Calling these endpoints/routes is wrong, it's really just a single endpoint/route with query string arguments